src/bluetooth.py

```diff
@@ -188,7 +188,7 @@ def services_resolved(self):
             # Get device firmware
             self.battery = int(battery_level.read_value()[0])
 
-        self.services_done()
```

Comment:
This will cause the battery level and the firmware version not to be displayed. services_done is defined by the connection thread in window.py

Reply:
I have experienced this while using this patch. The reason this was removed is that the "self" object here is not the same when called from the daemon. It does not seem to have a services_done object, causing the whole thing to crash.